packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -35,22 +42,71 @@ import type { TabComponent } from 'design/SlideTabs/SlideTabs';
 import type { ResourceType, ResourceLocation } from '../resource-lists';
 import type { AgentStepProps } from '../types';
 import type { State } from '../useDiscover';
+import type { AuthType } from 'teleport/services/user';
 
 export default function Container(props: AgentStepProps) {
   const ctx = useTeleport();
-  return <SelectResource nextStep={props.nextStep} />;
+  const { clusterId, isLeafCluster } = useStickyClusterId();
```

Comment:
we don't need this hook, we can only add new resources on the root cluster, we can get the clusterId from `cfg.proxyCluster` like it does inside the hook